scripts/migrate-black.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# check out every commit added by the current branch, blackify them,
+# and generate diffs to reconstruct the original commits, but then
+# blackified
+import os
+import sys
+from subprocess import check_output, run, Popen, PIPE
+
+
+def git(*args):
+    return check_output(["git"] + list(args)).decode("utf8").strip()
+
+
+def blackify(base_branch, black_command):
+    current_branch = git("branch", "--show-current")
+
+    if not current_branch or base_branch == current_branch:
+        print("You need to check out a feature brach to work on")
```

Comment:
Generally logging message I recommend using a logger from the logging module and writing to stderr - But we can clean that up in a follow up diff.